src/components/HallOfFameButton.js

```diff
@@ -0,0 +1,15 @@
+const MenuButton = (firstLetterButtonInnerText, restOfTextButtonInnerText) => {
+
+    const button = document.createElement('button');
+    button.classList.add('menuButton');
+
+    const firstLetter = document.createElement('span');
+    firstLetter.classList.add('menuButton__text');
+    firstLetter.innerText = restOfTextButtonInnerText;
+    
+    button.appendChild(firstLetter);
+
+    return button;
+}
+
+export default MenuButton;
```

Comment:
firstLetterButtonInnerText - tego argumentu już nie używasz, więc można go wywalić.
restOfTextButtonInnerText - nazwa tego argumentu już nie jest poprawna więc proponuję zmienić ją na "buttonText" albo coś takiego
firstLetter - nazwa tej zmiennej też już nie pasuje po zmianach, więc też proponuję ją zmienić :)
skoro komponent to MenuButton to proponuję też zmienić nazwę pliku na taką :D